src/coreclr/tools/Common/TypeSystem/IL/Stubs/ComparerIntrinsics.cs

```diff
@@ -2,7 +2,6 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 using System.Collections.Generic;
-
```

Comment:
Nit: we separate System namespaces from internal namespaces with a blank line. It's intentional.